packages/saltcorn-data/models/table.ts

```diff
@@ -1875,13 +1877,8 @@ class Table implements AbstractTable {
         v,
         id,
         user,
-        false,
-        resultCollector,
-        undefined,
-        undefined,
-        undefined,
-        undefined,
-        extraArgs
+        { noTrigger: false, extraArgs },
+        resultCollector
```

Comment:
in this case, resultCollector should also be inside the object: https://github.com/saltcorn/saltcorn/blob/93354594979180c8782a669b4d1d9614c0fe89f3/packages/saltcorn-data/models/table.ts#L1427

Reply:
changed it